LottieDropper/Dropbox/DropboxBrowserViewModel.m

```diff
@@ -46,29 +49,43 @@ - (void)initializeDropboxClient: (void (^)(void)) done  {
 		NSLog(@" No Dropbox user client");
 		return;
 	}
-	[[self.client.filesRoutes listFolder:@""]
-	 setResponseBlock:^(DBFILESListFolderResult *response, DBFILESListFolderError *routeError, DBRequestError *networkError) {
-		 if (response) {
-			 [self.entries addObjectsFromArray:response.entries];
+    
+    Connectivity *connectivity = [[Connectivity alloc]init];
+    if ([connectivity IsConnectionAvailable]) {
```

Comment:
Hier moet je checken of er een file is en die dan tonen. Dit verhelpt het op te lossen probleem niet maar bypassed het.